movie_library/views/actor_controller.py

```diff
@@ -1,27 +1,74 @@
+from django.shortcuts import get_object_or_404
+from rest_framework import status
 from rest_framework.viewsets import ViewSet
 from rest_framework.response import Response
+from ..utils.custome_premissions import CustomPermissions
+from ..models.movie import Movie
 from ..models.actor import Actor
 from ..utils.serializers.actor_serializer import ActorSerializer
+from ..utils.serializers.actor_movies_serializer import ActorMoviesSerializer
+from ..models.movie_actor import MovieActor
 
 
 class ActorViewSet(ViewSet):
+    permission_classes = [CustomPermissions]
 
     def list(self, request):
         actors = Actor.objects.all()
         result = ActorSerializer(actors, many=True)
         return Response(data=result.data, status=200)
 
     def create(self, request):
-        pass
+        data = request.data
+        movies = data.get('movies')
+        actor = Actor(first_name=data.get('first_name'), last_name=data.get('last_name'), gender=data.get('gender'))
+        actor.save()
+        bulk_list = list()
+        query = Movie.objects.all()
```

Comment:
it is better to use something like this 
Movie.objects.filter(id__in=movies_ids).all()
and check if length are equal